Makefile

```diff
@@ -651,6 +651,7 @@ $(FOMANTIC_DEST): $(FOMANTIC_CONFIGS) | node_modules
 	cp -f web_src/fomantic/theme.config.less node_modules/fomantic-ui/src/theme.config
 	cp -rf web_src/fomantic/_site/* node_modules/fomantic-ui/src/_site/
 	npx gulp -f node_modules/fomantic-ui/gulpfile.js build
+	perl -p -i -e 's#.*to-be-removed.*\n##g' web_src/fomantic/build/semantic.css
```

Comment:
There's now way around this if we want avatar images to render in the HTML-defined size besides setting invalid values for fomantic's variables which I think is even worse than removing them (which this here does).

Reply:
Could we use other tools rather than a new dependency?

Reply:
Hmm could also use `sed` I guess which we already depend on.

Reply:
Thought `sed` can not remove newlines and fomantic build depends on node, so I'll make it a node script.

Reply:
Script added.

Reply:
Actually, removed again. I think it's not yet warranted to do this kind of processing for something that's not really an issue to override.